tests/conftest.py

```diff
@@ -368,7 +410,16 @@ def check_new_tab(selenium):
     assert "about:newtab" in selenium.current_url
 
 
-@given("Firefox is launched enrolled in an Experiment", target_fixture="selenium")
-def _selenium(selenium):
+@given(
+    "Firefox is launched enrolled in an Experiment with custom search", target_fixture="selenium"
+)
+def setup_browser(selenium, setup_search_test):
     selenium.implicitly_wait(5)
```

Comment:
Why the 5s sleep?

Reply:
This isn't necessarily needed but I added it in case I needed it. I can remove it if you think it should be removed until needed

Reply:
🤷 Leave it as is... that way you can reduce it to 3s in Q2 and claim we sped up the test by 40%.